idmega2000/visitrail
# chore(setup): add doc strings to code

**What does this PR do?**
It adds pylint and doc strings to the project

**Description of Task to be completed?**
- add doc strings to existing code
- add pylint to project

**How should this be manually tested?**
- Fetch the branch `git fetch ch-setup-pylint-167337607`
- Check out into the branch `ft-user-authentication-166804623`
- run python manage.py migrate
- run python manage.py runserver
- The codebase should be well documented

**Any background context you want to provide?**
NIL

**What are the relevant pivotal tracker stories?**

[167337607](https://www.pivotaltracker.com/story/show/167337607)

**Screenshots (if appropriate)**

NIL
